docs/testing/test_quickstart.txt

```diff
@@ -0,0 +1,15 @@
+PREREQUISITES
+
+$cargo install cargo-nextest --locked
+
+install git-lfs in order to download test binaries (lightwalletd, zcashd, zcash-cli) to test against
+
+
+EXPLANATION
+
+there are two categories of integration tests: those running against zcashd's regtest mode (integration tests) and those running against lightwalletd's darkside mode (darkside tests)
+
+each of these modes is described in more detail in another document in this folder
```

Comment:
references file that has been removed (integration_tests.txt). could add instructions here or update and move integration_tests.txt to docs/testing?

Reply:
clarified instructions. the deleted file has been incorporated